samcli/commands/sync/command.py

```diff
@@ -244,15 +224,9 @@ def do_cli(
 
     confirmation_text = SYNC_CONFIRMATION_TEXT
```

Comment:
nit: then we can remove this line here by directly using `SYNC_CONFIRMATION_TEXT` below